plugins/connection/qubes.py

```diff
@@ -114,42 +114,42 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
         """Run specified command in a running QubesVM """
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
 
-        display.vvvv("CMD IS: %s" % cmd)
+        display.vvvv(f"CMD IS: {cmd}")
 
         rc, stdout, stderr = self._qubes(cmd)
 
-        display.vvvvv("STDOUT %r STDERR %r" % (stderr, stderr))
+        display.vvvvv(f"STDOUT {stderr!r} STDERR {stderr!r}")
```

Comment:
This looks like a bug, it's printing stderr twice. (Not related to this PR though.)

Reply:
I'd rather fix the bug first

Reply:
addressed in #9334

Reply:
rebased!